### DISCUSSION TITLE
Galaxy brain

### DISCUSSION THREAD

**Discussion by sethshivam11** [drknzz/GitHub-Achievements#349] [General]
I want to achieve the galaxy brain badge

**Comment by rehan509** [comment]
To get the Galaxy Brain achievement you need to:

answer at least 2 discussions and
have the OP mark your answers to the questions.
Hope that helps and you get Galaxy Brain soon ;)

**Reply by sethshivam11** [reply]
@rehan509  So, what else i have to do?
